### ISSUE TITLE
Can we get file size and date info?

### ISSUE THREAD

**Issue by ericloyd** [NagiosEnterprises/ncpa#581]
Would love to be able to get file size and/or last modification dates for files.  I know that's possible with some plugins at this point, but it would be nice to get that inherently from the agent.  Add that to your list of future possibilities, please?

**Comment by jomann09** [comment]
I like the idea of this, and I think there was a request for watching folders (checking for file changes, etc) which could definitely be useful too. I'll have to think about how to add this, there likely won't be a way to list all the files because that could easily be a security risk.

**Comment by ericloyd** [comment]
Understood.  Maybe having a configuration that says what folders are allowed, and making sure that c:\this\folder\is\allowed\..\..\..\..\windows wouldn't be allowed.  In other words, if a user wants to expose their filesystem to the agent, let them, but don't expose more than what they allow.

**Comment by jomann09** [comment]
Yeah I think that's a good idea, it'll have to be something configurable.
